### PR TITLE
Change affiliation of C Tantithamthavorn and P Thongtanunam

### PR DESCRIPTION
**Pull request by klainfo**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -1980,7 +1980,7 @@ Chad Jenkins,University of Michigan,http://ohseejay.org,dp5LnVAAAAAJ
 Chagit Attiya,Technion,http://hagit.net.technion.ac.il,eoMK92MAAAAJ
 Chai Kiat Yeo,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=ASCKYEO&CategoryDescription=interactivedigitalmedia,NOSCHOLARPAGE
 Chai Quek,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=ASHCQUEK,NOSCHOLARPAGE
-Chakkrit Tantithamthavorn,University of Adelaide,http://chakkrit.com,idShgcoAAAAJ
+Chakkrit Tantithamthavorn,Monash University,http://chakkrit.com,idShgcoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:1983]: Web page indicates Adelaide, not Monash.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Closing for lack of response; please feel free to re-open.

File: csrankings.csv (hunk 1 above)
Comment (klainfo) [reply csrankings.csv:1983]: @emeryberger Sorry for the delay. 

The website is updated now.

http://chakkrit.com
https://research.monash.edu/en/persons/chakkrit-tantithamthavorn

**Comment by klainfo** [comment]
@emeryberger Updated!

**Review by emeryberger — APPROVED** [review]

**Pull request merged**